myapp/app/views/tasks/index.html.erb

```diff
@@ -2,9 +2,15 @@
 
 <table class="table table-hover">
   <thead>
-    <tr>
-      <th scope="col"><%= t "title" %></th>
-      <th scope="col"><%= t "description" %></th>
+    <tr class="small">
+      <th scope="col"><%= t 'title' %></th>
+      <th scope="col"><%= t 'description' %></th>
+      <th scope="col">
+        <%= link_to_unless(params[:sort].eql?('created_at'), raw(t 'created_at' ) + raw('<i class="bi bi-sort-up"></i>'), root_path(:sort => 'created_at')) %>
```

Comment:
I'm sorry for leaving this comment after the merge. Just for your information, using `raw` and `html_safe` will be flagged by Rubocop in production code.
https://akinov.hatenablog.com/entry/2017/05/25/112310

I'm sorry. I don't check the behavior but this way would be better. 
```ruby
<%= link_to_unless(params[:sort] == 'created_at', root_path(sort: 'created_at')) do %>
  <%= t('created_at') %>
  <i class="bi bi-sort-up"></i>
<% end %>
```

Reply:
Thanks for the suggestion! I will fix it so in step 13.